### ISSUE TITLE
Predict Fixture Dates Issue

### ISSUE THREAD

**Issue by keith0811** [kochlisGit/ProphitBet-Soccer-Bets-Predictor#50]
https://prnt.sc/NRGxMBwT6D5v

If the match date is 13 Nov, however when we put in 13 Nov, it show error.
Parsing Error: Failed to parse fixture table, because date "Nov 13 ~" was not found. Perhaps the selected date is incorrect or the HTML of Footystats code has changed.

How should we input the date?

**Comment by KnightRider37** [comment]
Yes, you have to play around with the code in "parser.py" file.

you can try  
for i in range(1, (num_matches * 5) + 1, 5):
            odds_1.append(odd_lines[i].split('>')[1].split('<')[0].replace('\n', ''))
            odds_x.append(odd_lines[i + 1].split('>')[1].split('<')[0].replace('\n', ''))
            odds_2.append(odd_lines[i + 2].split('>')[1].split('<')[0].replace('\n', ''))
        return odds_1, odds_x, odds_2

instead of what's in the file now. It's not perfect, but works a bit better.

**Comment by keith0811** [comment]
Thanks. I realise we can manually change the home/away team and its odds.

**Comment by KnightRider37** [comment]
> Thanks. I realise we can manually change the home/away team and its odds.

Yes, it can be done for both teams and also for the odds, but it's very time-consuming.
